src/routes/(app)/order/[id]/payment/[paymentId]/ticket/+page.svelte

```diff
@@ -0,0 +1,302 @@
+<script lang="ts">
+	import Picture from '$lib/components/Picture.svelte';
+	import PriceTag from '$lib/components/PriceTag.svelte';
+	import Trans from '$lib/components/Trans.svelte';
+	import { useI18n } from '$lib/i18n.js';
+	import { invoiceNumberVariables } from '$lib/types/Order.js';
+	import { fixCurrencyRounding } from '$lib/utils/fixCurrencyRounding';
+	import { sum } from '$lib/utils/sum.js';
+	import { sumCurrency } from '$lib/utils/sumCurrency.js';
+	import { differenceInMinutes } from 'date-fns';
+	import { marked } from 'marked';
+	import DEFAULT_LOGO from '$lib/assets/bebop-light.svg';
+
+	export let data;
+
+	const { t, locale, textAddress, countryName } = useI18n();
+
+	const finalInvoice = data.payment.status === 'paid';
+	const invoiceNumber = t(
+		finalInvoice ? 'order.receipt.invoiceNumber' : 'order.receipt.proformaInvoiceNumber',
+		invoiceNumberVariables(data.order, data.payment)
+	);
+	const identity = data.sellerIdentity;
+
+	const differentAddress =
+		data.order.shippingAddress &&
+		data.order.billingAddress &&
+		textAddress(data.order.shippingAddress) !== textAddress(data.order.billingAddress);
+
+	const currency = data.order.currencySnapshot.main.totalPrice.currency;
+
+	const totalPriceWithoutDiscount = {
+		currency,
+		amount: data.order.currencySnapshot.main.discount
+			? fixCurrencyRounding(
+					data.order.currencySnapshot.main.totalPrice.amount +
+						data.order.currencySnapshot.main.discount.amount,
+					currency
+			  )
+			: data.order.currencySnapshot.main.totalPrice.amount
+	};
+
+	const discountPercentage = data.order.currencySnapshot.main.discount?.amount
```

Comment:
This is not a percentage (ie. value from 0 to 100), rather a factor (0-1).

How about `discountFactor`?